pkg/virtctl/imageupload/imageupload.go

```diff
@@ -133,7 +134,8 @@ func NewImageUploadCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 	cmd.Flags().StringVar(&size, "size", "", "The size of the DataVolume to create (ex. 10Gi, 500Mi).")
 	cmd.Flags().StringVar(&storageClass, "storage-class", "", "The storage class for the PVC.")
 	cmd.Flags().StringVar(&accessMode, "access-mode", "ReadWriteOnce", "The access mode for the PVC.")
-	cmd.Flags().BoolVar(&blockVolume, "block-volume", false, "Create a PVC with VolumeMode=Block (default Filesystem).")
+	cmd.Flags().StringVar(&volumeMode, "volume-mode", "Filesystem", "The volume mode for the PVC.")
```

Comment:
can we document the available volume modes in the description here? I know it should be obvious, but it helps not making people guess the exact terms being used and perhaps the case of the value (like `Block` instead of `block`)

Reply:
done